cranelift-codegen/meta/src/cdsl/ast.rs

```diff
@@ -95,6 +96,55 @@ impl DefPool {
 pub struct DefIndex(u32);
 entity_impl!(DefIndex);
 
+/// A definition which would lead to generate a block creation.
+#[derive(Clone)]
+pub struct Block {
```

Comment:
Can you mark this struct and the one below pub(crate) instead, please? (If that means transitively marking more stuff pub(crate), please do so.)